hockey/filterFunctions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+function getInfoFilter ($query){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1617
+    $query
```

Comment:
This has the potential for SQL injection (i.e. something from the `$_REQUEST` is being passed directly into the db without being sanitized. Your call on whether or not you care (I realize this is just a practice project) but I want to make sure you're aware of it.